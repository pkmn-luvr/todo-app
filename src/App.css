.App {
    max-width: 600px;
    margin: 40px auto;
    padding: 20px;
    background-color: #fff;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
  }
  