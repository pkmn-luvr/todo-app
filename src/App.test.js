import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App Component Tests', () => {
  // Smoke
  test('renders without crashing', () => {
    render(<App />);
  });


// Snapshot
  test('matches snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
