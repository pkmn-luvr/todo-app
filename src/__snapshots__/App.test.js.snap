// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Tests matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <form>
        <label
          for="new-todo"
        >
          Add a new task:
        </label>
        <input
          id="new-todo"
          placeholder="Add a new task"
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Add Todo
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
